### PR TITLE
Created new bill home page

### PR DESCRIPTION
**Pull request by Shibainuinuinu**

### CODE REVIEW

File: assets/main.css [hunk 1]
```diff
@@ -12,4 +12,42 @@ body {
 .tester {
     color: red;
     font-size: 3em;
-}
\ No newline at end of file
+}
+
+
+/* Start RSB-2 */
```
Comment (Cosze) [inline assets/main.css:18]: Should be a more descriptive name for what these styles are for e.g. "billing home page"

File: components/newBillHomePageButtons.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default function newBillHomePageButtons({text, iconName}){
+    return(
```
Comment (Cosze) [inline components/newBillHomePageButtons.jsx:2]: Should be a space beside the return like `return (`

File: components/newBillHomePageButtons.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export default function newBillHomePageButtons({text, iconName}){
```
Comment (Cosze) [inline components/newBillHomePageButtons.jsx:1]: There should be a space between the deconstructed fields and after the parameter definitions like `({ text, iconName }) {`

File: pages/newBillHomePage.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import Button from "../components/newBillHomePageButtons"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCamera } from '@fortawesome/free-solid-svg-icons'
+
+export default function newBillHomePage(){
+    return(
```
Comment (Cosze) [inline pages/newBillHomePage.js:6]: Same feedback as above

File: assets/main.css [hunk 5]
```diff
@@ -12,4 +12,42 @@ body {
...
     color: red;
     font-size: 3em;
-}
\ No newline at end of file
+}
+
+
+/* Start new bill home page */
+
+.newBillHomePageButton{
```
Comment (Cosze) [inline assets/main.css:20]: Can you add a space after the css selectors?

File: assets/main.css [hunk 6]
```diff
@@ -12,4 +12,42 @@ body {
...
+    border: 1px solid gray;
+    color: gray;
+    border-radius: 0.5em;
+    padding: 15px;
+    width: 250px;
+}
+.newBillHomePageButton:hover{
```
Comment (Cosze) [inline assets/main.css:27]: Same as above

File: assets/main.css [hunk 7]
```diff
@@ -12,4 +12,42 @@ body {
...
+    border: 1px solid rgb(255, 136, 0);
+    color: rgb(255, 136, 0);
+}
+
+.newBillHomePageButtonText{
```
Comment (Cosze) [inline assets/main.css:32]: Same as above

File: assets/main.css [hunk 8]
```diff
@@ -12,4 +12,42 @@ body {
...
+    display: flex;
+    justify-content:space-between;
+    align-items: center;
+}
+
+.newBillHomePageContainer{
```
Comment (Cosze) [inline assets/main.css:38]: Same as above

File: components/newBillHomePageButtons.jsx [hunk 9]
```diff
@@ -0,0 +1,10 @@
+export default function newBillHomePageButtons({ text, iconName }){
```
Comment (Cosze) [inline components/newBillHomePageButtons.jsx:1]: Please add a space between `({ text, iconName })` and the curly brace like `({ text, iconName }) {`

File: pages/newBillHomePage.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
+import Button from "../components/newBillHomePageButtons"
+
+export default function newBillHomePage(){
```
Comment (Cosze) [inline pages/newBillHomePage.js:3]: please add a space after the parenthesis like `newBillHomePage() {`

**Review by Cosze — APPROVED** [review]
LGTM thanks for making the changes 🚢

File: package-lock.json
Comment (kathyye15) [inline package-lock.json:1]: please remove the package lock changes as well!

**Review by kathyye15 — APPROVED** [review]

**Pull request merged**